### PR TITLE
Handle the base exception from the requests library

### PR DESCRIPTION
**Pull request by mgdaily**
We were handling one of the subclasses, and some errors slipped through, causing missed reductions.

Also update our deploy to use in-cluster URLs to speed up the DNS resolution and make us more resilient to DNS issues.

Sorry for all of the whitespace changes, but it looks like we had some extra end-of-line whitespace in the changelog.

### CODE REVIEW

File: helm-chart/banzai/values-prod.yaml [hunk 1]
```diff
@@ -6,20 +6,20 @@
 # will be started when the CPU usage rises above the configured threshold.
 horizontalPodAutoscaler:
   enabled: true
-  minReplicas: 15
-  maxReplicas: 15
+  minReplicas: 25
```
Comment (jchate6) [inline helm-chart/banzai/values-prod.yaml:9]: We should probably talk about how to introduce autoscaling someday when you have the bandwidth...

File: helm-chart/banzai/values-prod.yaml [hunk 2]
```diff
@@ -6,20 +6,20 @@
...
   targetCPUUtilizationPercentage: 50
-  minLargeReplicas: 7
-  maxLargeReplicas: 7
+  minLargeReplicas: 12
+  maxLargeReplicas: 12
 
 image:
   repository: ghcr.io/lcogt/banzai
-  tag: "1.18.1"
+  tag: "1.18.2"
```
Comment (jchate6) [inline helm-chart/banzai/values-prod.yaml:17]: Should this match the version number above in the changes? That's currently 1.18.3...

**Review by jchate6 — APPROVED** [review]
I don't really understand the reason for the banzai changes with urls, but I trust that you do.

I have an actual question about versioning, but again, I might be missing things, so I'm clicking approve.

File: helm-chart/banzai/values-prod.yaml (hunk 1 above)
Comment (cmccully) [reply helm-chart/banzai/values-prod.yaml:9]: Do we need to set a max replicas here? Or just assume if we only set a min, they are equal?

Comment (mgdaily) [reply helm-chart/banzai/values-prod.yaml:9]: > We should probably talk about how to introduce autoscaling someday when you have the bandwidth...

Jashan did prototype this at one time. https://github.com/LCOGT/banzai/pull/359/files

Autoscaling based on queue size _sounds_ great Joey, but I am very hesitant to commit my time to this as a development priority. I think it is relatively high effort to low gain, given that we can simply run more pods than we need most of the time.

I would love if someone other than me could spearhead that effort.

Comment (mgdaily) [reply helm-chart/banzai/values-prod.yaml:9]: > Do we need to set a max replicas here? Or just assume if we only set a min, they are equal?

Yes we need to. https://github.com/LCOGT/banzai/pull/359/files

This has always been the case.

File: helm-chart/banzai/values-prod.yaml (hunk 2 above)
Comment (mgdaily) [reply helm-chart/banzai/values-prod.yaml:17]: Good catch.

File: helm-chart/banzai/values-prod.yaml (hunk 1 above)
Comment (cmccully) [reply helm-chart/banzai/values-prod.yaml:9]: Yeah. Ok. github split the file and I missed the next line.

**Pull request merged**